theatre/studio/src/propEditors/simpleEditors/ImagePropEditor.tsx

```diff
@@ -0,0 +1,81 @@
+import type {PropTypeConfig_Image} from '@theatre/core/propTypes'
+import React, {useCallback, useEffect} from 'react'
+import styled from 'styled-components'
+import type {ISimplePropEditorReactProps} from './ISimplePropEditorReactProps'
+
+const Wrapper = styled.label`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  ${({empty}) => empty && `border: 1px dashed currentColor;`}
+  box-sizing: border-box;
+`
+
+// file input
+const Input = styled.input.attrs({type: 'file', accept: 'image/*'})`
```

Comment:
How do you remove an image?

Reply:
Well, that's also a discussion for the schema, like how we allow for an empty state, which is also something new as far as prop types go, but for now I added a remove button that just sets the id to ''.